test/integration/sitemap/sitemap_generator_test.rb

```diff
@@ -111,22 +144,37 @@ def test_links_should_include_timestamps
   end
 
   def test_links_should_include_priorities
-    generator = SitemapGenerator.new(search_server.content_indices)
+    generator = SitemapGenerator.new(index_names: ['govuk_test'])
+    add_sample_documents(
+      [
+        {
+          "title" => "Cheese in my face",
+          "description" => "Hummus weevils",
+          "format" => "answer",
+          "link" => "/an-example-answer",
+          "indexable_content" => "I like my badger: he is tasty and delicious",
+          "public_timestamp" => "2017-07-01T12:41:34+00:00"
+        },
+      ],
+      index_name: 'govuk_test'
+    )
 
     sitemap_xml = generator.sitemaps
 
-    priorities = Nokogiri::XML(sitemap_xml[0])
-      .css("url > priority")
+    pages = Nokogiri::XML(sitemap_xml[0])
+      .css("url")
+      .select { |item| item.css("loc").text == "http://www.dev.gov.uk/an-example-answer" }
 
-    assert_equal 6, priorities.count
+    assert_equal 1, pages.count
+    assert_equal "0.5", pages[0].css("priority").text
```

Comment:
This assertion is a bit confusing because it looks like it's the priority of the test in the setup, whereas it's actually the priority of the homepage.

I deliberately didn't add an assertion of a real priority in this test because I didn't want to make the integration tests depend on a slightly arbitrary number. The unit tests do some more in-depth testing of the actual value.

Reply:
@suzannehamilton I have changed the assertion to check the priority is within a range between 0-10. This will make it a little bit less brittle